Update README.md (#53)
opengeospatial/geoparquet: README.md

## About

This repository defines how to store geospatial [vector data](https://gisgeography.com/spatial-data-types-vector-raster/) (point, 
lines, polygons) in [Apache Parquet](https://parquet.apache.org/), a popular columnar storage format for tabular data - see 

> [what is parquet?](https://parquet.apache.org/docs/overview/) for more background. Our goal is to standardize how 

Reviewer: ```suggestion
the official [parquet overview](https://parquet.apache.org/docs/overview/) and the excellent [what is parquet?](https://databricks.com/glossary/what-is-parquet) explanation 
for more background. Our goal is to standardize how 
```

> [what is parquet?](https://parquet.apache.org/docs/overview/) for more background. Our goal is to standardize how 

Reviewer: [this vendor explanation](https://databricks.com/glossary/what-is-parquet) for more on what that means. Our goal is to standardize how

> [what is parquet?](https://parquet.apache.org/docs/overview/) for more background. Our goal is to standardize how 

Reviewer: ```suggestion
[this vendor explanation](https://databricks.com/glossary/what-is-parquet) for more on what that means. Our goal is to standardize how 
```
